src/app/g3w-ol/controls/screenshotcontrol.js

```diff
@@ -9,6 +10,16 @@ function ScreenshotControl(options = {}) {
   options.label = options.label || "\ue90f";
   options.toggled = false;
   OnClickControl.call(this, options);
+  // Listen when a new externalLayer is add to map
+  GUI.getService('map').onafter('loadExternalLayer', (layer)=> {
+    this.layers.push(layer);
```

Comment:
the list of available layers should be retrieved (and maintained) by a global layers store